packages/svelte-scoped/preprocessor/vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineProject } from 'vitest/config'
+
+export default defineProject({
+  test: {
+    globals: true,
```

Comment:
I would like to avoid have `globals` if possible

Reply:
Sure, I've removed that out.